angular-spa-ui/js/admin-app/admin-app.js

```diff
@@ -25,3 +25,19 @@ function configCb($stateProvider, $urlRouterProvider) {
     $urlRouterProvider
         .otherwise('content/publication/');
 };
+
+function bootstrap($rootScope, bootstrap, commonService) {
+    bootstrap.checkRegistration()
+        .then(function (responce) {
+            commonService.successValidationAction(responce.userData, responce.scope);
```

Comment:
Normally, response should contains a data, not a scope and other application entities.